controllers/chat/group/removeGroupMember.js

```diff
@@ -24,32 +34,35 @@ const removeGroupMember = async (req, res) => {
       const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
         await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
 
-      const client = new StreamChat(
-        Environment.GETSTREAM_API_KEY,
-        Environment.GETSTREAM_API_SECRET
-      );
-      const currentChannel = await client.channel(CHANNEL_TYPE_STRING.GROUP, channelId);
-
       const textOwnUser = `You removed ${targetUserModel.username} from this group`;
       const textTargetUser = `${ownUser.username} removed you from this group`;
       const textDefaultUser = `${ownUser.username} removed ${targetUserModel.username} from this group`;
-      const members = betterChannelMember.map((member) => member.user_id);
 
-      await currentChannel.sendMessage({
-        text: textDefaultUser,
-        own_text: textOwnUser,
-        other_text: textTargetUser,
-        other_system_user: userId,
-        better_type: 'remove_member_from_group',
-        type: 'system',
-        user_id: userId,
-        only_to_user_show: userId,
-        disable_to_user: false,
-        is_from_prepopulated: true,
-        system_user: userId,
-        isSystem: true,
-        members: members
-      });
+      await currentChannel.sendMessage(
+        {
+          text: textDefaultUser,
+          own_text: textOwnUser,
+          other_text: textTargetUser,
+          other_system_user: targetUserModel.user_id,
+          better_type: 'remove_member_from_group',
+          type: 'system',
+          user_id: userId,
+          only_to_user_show: userId,
+          disable_to_user: false,
+          is_from_prepopulated: true,
+          system_user: userId,
+          isSystem: true,
+          members: all_members
+        },
+        {
+          skip_push: true
+        }
+      );
+
+      await BetterSocialCore.fcmToken.sendGroupChatNotification(
+        targetUserModel.user_id,
+        textTargetUser
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here seem fine as they add more context to the messages being sent and improve the handling of notifications. However, it's important to ensure that the `sendGroupChatNotification` function handles errors properly, especially since it involves external services (Firebase Cloud Messaging). If an error occurs while sending the notification, it should be caught and handled appropriately to prevent the entire operation from failing. If this isn't already implemented in the `sendGroupChatNotification` function, consider adding error handling there.


<!-- This is an auto-generated comment by OSS CodeRabbit -->